lib/bamboo/phoenix.ex

```diff
@@ -51,6 +51,49 @@ defmodule Bamboo.Phoenix do
           |> render("text_email.text")
         end
       end
+
+  ## Full HTML Layout Example
+
+      # web/email.ex
+      defmodule Changelog.Email do
```

Comment:
Could you change all instance of `Changelog` to `MyApp`? That's the default namespace used in the Bamboo docs